### ISSUE TITLE
wcp

### ISSUE THREAD

**Issue by letypequividelespoubelles** [Consensys/besu-sequencer-plugins#1260]

**Comment by OlivierBBB** [comment]
## Results

Success

<img width="659" alt="image" src="https://github.com/user-attachments/assets/b842fb7d-2540-49d4-996b-2b6c329ca7b6">

## Local setup
```bash
tracer:
* branch: arith-dev
* commit: b649515d0022a6e0dbac3bebe33320ad282eca7a

constraints:
* branch: master
* commit: 8cbf5b63fcae5e82a8312132b1b414be7e1febd1
```

**Comment by letypequividelespoubelles** [comment]
closed as not reproduced with 0.6.0-rc5
